README.md

```diff
@@ -1,5 +1,30 @@
-# Equivalence Classes
+# Equivalence Classes for LOINC Terms (LOINC version 2.66)
 
-These files are associated with the AMIA 2019 Workshop W17: FHIR forms, flowsheets, and other EHR functions for FHIR.
+For some purposes, such a flowsheet, it is useful to regard groups of
```

Comment:
Add "Clinical or EHR" to flowsheet to help clarify.

Reply:
Added.

Reply:
Added